### PR TITLE
Add a GH action to autopopulate readme pages

### PR DESCRIPTION
**Pull request by dougsillars**
## Description
Please include a summary of the change, motivation and context.

adding a GH action that reads al of the xRunBooks, and creates a list (with clickable links) for the main README. It also autocreates README files for each directory of xRUnBooks (AWS, Mongo, K8s, etc.)

This ensures that the readme lists are up to date.

### Testing
Tested the Py code locally, and have done several PRs in my fork to ensure that the lists are populating as expected.

### Checklist:
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published. 

### Documentation
Make sure that you have documented corresponding changes in this repository. 

The one new think is that all text in the main README that is not in the lust should now ONLY be updated in the readme_template file.  any changes made in the readme will be overwritten.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,43 +68,50 @@ These xRunBooks are included in every install.  Use them as is, or make a copy t
...
-1. Work with us on any of our [open issues](https://github.com/unskript/awesome-cloudops-automation/issues).
-2. Create a new Action. Read the [Action developer guide](https://github.com/unskript/Awesome-CloudOps-Automation/blob/master/.github/DEVELOPERGUIDE.md) for more details.
-3. Combine Actions into a xRunBook.  Your xRunBook will be featured on the ReadMe.  [What's a xRunBook?](https://github.com/unskript/Awesome-CloudOps-Automation/blob/master/.github/Anatomy-of-Runbook.md)
 
 
-
-### Contribution Tips:
-1. Use the [Docker environment](#get-started-with-docker), or our free [cloud sandbox](https://us.app.unskript.io/profiles/6c38d3da1cde7b3c0623d138f525a5508a3260c8) for testing your contribution.
-2. Join our [Slack Community](https://communityinviter.com/apps/cloud-ops-community/awesome-cloud-automation) so you can present yourself and get support.
+### How to Contribute?
```
Comment (abhishek-unskript) [inline README.md:114]: Are we moving this section (How to Contribute) out to somewhere else? Should we add a link instead?

File: generate_readme.ipynb [hunk 2]
```diff
@@ -0,0 +1,609 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "latter-teddy",
+   "metadata": {},
+   "source": [
+    "\n",
+    "# Generate Readme for Awesome Notebooks"
```
Comment (abhishek-unskript) [inline generate_readme.ipynb:9]: update :-)

File: .ipynb_checkpoints/generate_readme-checkpoint.ipynb [hunk 3]
```diff
@@ -0,0 +1,1058 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "latter-teddy",
+   "metadata": {},
+   "source": [
+    "<img width=\"10%\" alt=\"Naas\" src=\"https://landen.imgix.net/jtci2pxwjczr/assets/5ice39g4.png?w=160\"/>\n",
+    "\n",
+    "# Generate Readme for Awesome Notebooks"
```
Comment (abhishek-unskript) [inline .ipynb_checkpoints/generate_readme-checkpoint.ipynb:10]: update :-)

**Review by abhishek-unskript — COMMENTED** [review]
Please remove the .DS_store files (add to git ignore list?)

Also had a couple of questions (in the comments)

File: README.md (hunk 1 above)
Comment (dougsillars) [reply README.md:114]: The readme.md is built from the readme_template.md.  so it doesn't matter what is in the current readme, the action will overwrite the file.

These lines are in the tenmplate file, so will be in the readme on completion of the action

File: generate_readme.ipynb (hunk 2 above)
Comment (dougsillars) [reply generate_readme.ipynb]: updated

File: .ipynb_checkpoints/generate_readme-checkpoint.ipynb (hunk 3 above)
Comment (dougsillars) [reply .ipynb_checkpoints/generate_readme-checkpoint.ipynb]: removed these checkpoint files..

**Review by abhishek-unskript — APPROVED** [review]

**Pull request merged**